sn_networking/src/event.rs

```diff
@@ -1217,6 +1252,72 @@ impl SwarmDriver {
             }
         }
     }
+
+    /// Check among all chunk type records that we have, select those close to the peer,
+    /// and randomly pick one as the verification candidate.
+    #[allow(clippy::mutable_key_type)]
+    fn select_verify_candiates(&mut self, peer: NetworkAddress) -> Vec<NetworkAddress> {
```

Comment:
`select_verify_candiates` this naming is quite vague.
`select_verification_data_candidates` perhaps?